fuse-products/src/main/java/software/tnb/product/application/App.java

```diff
@@ -133,7 +133,7 @@ protected List<String> systemProperties() {
      *
      * @param arguments command to execute. "camel export --kamelets-version xxx --dir xxx --gav xxx integrationClass.java" are added automatically.
      */
-    protected void createUsingJBang(List<String> arguments) {
+    protected void createUsingJBang(List<String> arguments, String runtime) {
```

Comment:
you can get the product using `TestConfiguration.product()`

Reply:
ohhh yeah, I was looking for something related to runtime, not product :)